nicolas-martin-castellanos/tarea-1/magazine1/style.css

```diff
@@ -0,0 +1,130 @@
+@import url('https://fonts.googleapis.com/css2?family=Anton&family=Antonio:wght@700&family=Roboto+Condensed:wght@700&display=swap');
+*{
+    padding: 0px;
+    margin: 0px;
+}
+body{
+    background-image: url(./imagenes/fondo.png);
+    background-repeat: space;
+    background-position-y: bottom;
+}
+.box{
+    width: 100%;
+    height: 100vh;
+    background-color: rgba(0, 0, 0, 0.4);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.jordan{
+    z-index: 1;
```

Comment:
@NicolasMartin23 , please make sure all CSS properties are alphabetically sorted

Reply:
Send Magazine with CSS properties sorted alphabetically to PR.